code/mob/living/carbon/human/procs/Life.dm

```diff
@@ -933,13 +933,27 @@
 
 		if (breath.temperature > min(organHolder.left_lung ? organHolder.left_lung.temp_tolerance : INFINITY, organHolder.right_lung ? organHolder.right_lung.temp_tolerance : INFINITY) && !src.is_heat_resistant()) // Hot air hurts :(
 			//checks the temperature threshold for each lung, ignoring missing ones. the case of having no lungs is handled in handle_breath.
-			var/burn_damage = min((breath.temperature - (T0C+66)) / 3,10) + 6
-			TakeDamage("chest", 0, burn_damage, 0, DAMAGE_BURN)
+			var/lung_burn_left = min(max(breath.temperature - organHolder.left_lung?.temp_tolerance, 0), 10)
+			var/lung_burn_right = min(max(breath.temperature - organHolder.right_lung?.temp_tolerance, 0), 10)
+//			var/burn_damage = min((breath.temperature - (T0C+66)) / 3,10) + 6
```

Comment:
delete this

Reply:
that was what you were asking for, right?